.github/workflows/release-manifests.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Manifest
+
+# This workflow should be run after the integration is released into a new infrastructure-bundle image version
+# Update RELEASE_VERSION and INFRA_BUNDLE_VERSION on the Makefile before run.
+on:
+  workflow_dispatch:
```

Comment:
is this going to be triggered automatically once the infra bundle has been updated?

Reply:
nope, the process is manual, also still need to bump the versions on the makefile variables.